notebooks/6_calculate_statistics.ipynb

```diff
@@ -0,0 +1,873 @@
+{
```

Comment:
<p>typo: "Nan" should be "NaN"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stijnvanhoey/pandas-getting-started-tutorials/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>